core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/api/toDataFrame.kt

```diff
@@ -38,20 +39,37 @@ import kotlin.reflect.jvm.javaField
 import kotlin.reflect.typeOf
 
 private val valueTypes = setOf(
+    Any::class,
+    Unit::class,
+    Char::class,
+    UByte::class,
+    UShort::class,
+    UInt::class,
+    ULong::class,
     String::class,
     Boolean::class,
     kotlin.time.Duration::class,
     kotlinx.datetime.LocalDate::class,
     kotlinx.datetime.LocalDateTime::class,
     kotlinx.datetime.Instant::class,
+    kotlinx.datetime.TimeZone::class,
+    kotlinx.datetime.DateTimePeriod::class,
+    kotlinx.datetime.DateTimeUnit::class,
 )
 
+/**
+ * Should be aligned with `ConeKotlinType.isValueType()` in
+ * plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/impl/api/toDataFrame.kt
+ */
+@PublishedApi
 internal val KClass<*>.isValueType: Boolean
```

Comment:
yess! This is great, a stable definition of what we consider a value type :D love it